.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,91 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Exercises check
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+
+jobs:
+  housekeeping:
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Download & Install dependencies
+      run: |
+        ./bin/fetch-configlet
+        python -m pip install --upgrade pip
+        pip install flake8 
+        git clone https://github.com/exercism/problem-specifications spec
+        pip download -r requirements-generator.txt -d pydeps
+        pip install --no-index pydeps/*
+    - name: Upload dependency artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: dependencies
+        path: pydeps
+        
+    - name: Check readmes
```

Comment:
Looking at the [output](https://github.com/exercism/python/pull/2200/checks?check_run_id=653910619#step:6:1) for this step, it doesn't seem to be running correctly. There are some READMEs that report out-of-date, but the workflow doesn't fail. Additionally, not every exercise seems to be checked.

This may be a bug with the `check-readmes.sh` script, not this workflow.

Reply:
[Travis build has the same problem](https://travis-ci.org/github/exercism/python/jobs/684374749#L239). I'll make a separate issue to fix this script.

Reply:
Should I resolve this as it is grew into new issue?